metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/_datahub_ol_adapter.py

```diff
@@ -14,10 +14,15 @@
 
 
 def translate_ol_to_datahub_urn(ol_uri: OpenLineageDataset) -> str:
-    namespace = ol_uri.namespace
-    name = ol_uri.name
-
-    scheme, *rest = namespace.split("://", maxsplit=1)
-
-    platform = OL_SCHEME_TWEAKS.get(scheme, scheme)
-    return builder.make_dataset_urn(platform=platform, name=name)
+    if not hasattr(ol_uri, "namespace") or not ol_uri.namespace:
```

Comment:
when does this happen? what should the type annotation of this function be changed to?

Reply:
this is no longer required and reverted back to the original